platforms/core-runtime/internal-instrumentation-processor/src/main/java/org/gradle/internal/instrumentation/extensions/property/PropertyUpgradeAnnotatedMethodReader.java

```diff
@@ -171,22 +172,17 @@ public Collection<Result> readRequest(ExecutableElement method, ReadRequestConte
         try {
             List<AccessorSpec> accessorSpecs = readAccessorSpecsFromReplacesEagerProperty(method, annotationMirror, context);
             List<CallInterceptionRequest> requests = new ArrayList<>();
+            Map<String, List<AccessorSpec>> groovyUpgradeAccessorSpecs = new LinkedHashMap<>();
```

Comment:
❔ Do we need a map because there might be several interception requests for the same property?

Reply:
Yes, exactly. One example is boolean. We have cases where we have `isX()` and `getX()` methods